script/script.js

```diff
@@ -0,0 +1,40 @@
+
+  $(document).ready(function(){
+  	
+  	var $pushtolocal = [];
+  	var $pulledfromlocal = localStorage.getItem('userinput');
+  	  	
+  	if ($pulledfromlocal != null){
+  		$('ul').append('<li><input type="checkbox" class="checkedbox"></input><span>' + $pulledfromlocal + '</span></li>'); /* tried to add 'id="local" so i could target this specifically to initialte the local storage clear when it's checkbox is clicked but it doenst work :(*/
```

Comment:
Because your $pulledfromlocal is a parse array, you'll need to iterate over it to print each item again:
```
if ($pulledfromlocal != null) {
    for (var index in $pulledfromlocal) {
        $('ul').append('<li><input type="checkbox" class="checkedbox"></input><span>' + $pulledfromlocal[index] + '</span></li>');
    }
}
```